azure-devops/azext_devops/dev/team/commands.py

```diff
@@ -111,3 +119,17 @@ def load_team_commands(self, _):
         g.command('enable', 'enable_extension', table_transformer=transform_extension_table_output)
         g.command('disable', 'disable_extension', table_transformer=transform_extension_table_output)
         g.command('search', 'search_extensions', table_transformer=transform_extension_search_results_table_output)
+
+    with self.command_group('devops wiki', command_type=wikiOps) as g:
+        g.command('create', 'create_wiki', table_transformer=transform_wiki_table_output)
+        g.command('list', 'list_wiki', table_transformer=transform_wikis_table_output)
+        g.command('show', 'show_wiki', table_transformer=transform_wiki_table_output)
+        g.command('delete', 'delete_wiki', table_transformer=transform_wiki_table_output,
+                  confirmation='Are you sure you want to delete this wiki?')
+
+    with self.command_group('devops wiki page', command_type=wikiOps) as g:
```

Comment:
do we plan to other page related things like 
metadata
revisions
linked work item

Reply:
Probably later.